src/library/baseexternalplaylistmodel.cpp

```diff
@@ -97,37 +97,42 @@ void BaseExternalPlaylistModel::setPlaylist(QString playlist_path) {
     }
 
     if (playlistId == -1) {
-        qDebug() << "ERROR: Could not get the playlist ID for playlist:" << playlist_path;
+        qWarning() << "ERROR: Could not get the playlist ID for playlist:" << playlist_path;
         return;
     }
 
-    QString playlistViewTable = QString("%1_%2").arg(m_playlistTracksTable,
-            QString::number(playlistId));
-
-    QStringList columns;
-    columns << "track_id";
-    columns << "position";
-    columns << "'' AS " + LIBRARYTABLE_PREVIEW;
-
-    QSqlQuery query(m_database);
-    FieldEscaper f(m_database);
-    QString queryString =
+    const auto playlistIdNumber =
```

Comment:
Personally, I think using `auto` instead of `QString` only saves 2 chars and makes it less apparent which type we're dealing with.

Reply:
Arguing about individual cases is the wrong approach. The length of 'QString' should not matter, it could be any type. The usage of auto must not depend on how many characters it saves.

More appropriate rule: `auto` is allowed if the resulting type is immediately apparent from the right-hand-side expression.

Reply:
In this case, it reduces redundancy, which is worth more than saving 2 characters ;)

Reply:
Ok, then I'm fine with this.